stablehlo/dialect/StablehloOps.td

```diff
@@ -50,6 +57,13 @@ include "stablehlo/dialect/StablehloAttrs.td"
 class StableHLO_ShapedInterfaceOp<string mnemonic, list<Trait> traits> :
     StableHLO_Op<mnemonic, traits # [DeclareOpInterfaceMethods<InferShapedTypeOpInterface,
     ["reifyReturnTypeShapes"]>]> {
+  let extraClassDeclaration = [{
+    // Relax the strict default implementation with one that allows
+    // for StableHLO-specific differences.
```

Comment:
I understand that it's not easy to get rid of the need to duplicate `isCompatibleReturnTypes` everywhere, and that's fine for now, but let's delete the repetitive comments. We can keep one comment in `CHLO_Op`, one comment in `StableHLO_Op`, but I think that the rest should go.

Reply:
Make senses! Only keep 1 comment in each td file.